client/src/PieChart.js

```diff
@@ -33,17 +33,15 @@ class PieChart extends React.Component {
     this.setState({
       year: event.target.value
     })
-    
   }
 
   fetchData() {
-    var myInit = { method: 'GET',
+    const myInit = { method: 'GET',
```

Comment:
```suggestion
    const options = { 
        method: 'GET',
```
Et ferme avec un retour à la ligne, pareil sur les autres myInit !